patterns/components/breadcrumb.php

```diff
@@ -1,7 +1,7 @@
-<nav class="breadcrumbs" role="navigation">
-  <ul class="breadcrumbs__list">
-    <?php foreach ($breadcrumbs as $breadcrumb): ?>
-      <li class="breadcrumbs__list-item font--secondary--xs upper dib"><a href="<?php echo $breadcrumb['link']; ?>" class="breadcrumbs__link"><?php echo $breadcrumb['name']; ?></a></li>
-    <?php endforeach; ?>
-  </ul>
-</nav> <!-- /.breadcrumbs -->
+<?php
+  if (function_exists('yoast_breadcrumb')) {
```

Comment:
I think we can successfully refactor the breadcrumb logic to not use Yoast. Let's discuss if you need help with this.